File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12577_:::_Hajj-e-Akbar/Main.cpp
using namespace std;
#include <cstdio>
#include <cstring>

int main()
{
	char comando[6];
	scanf("%s", comando);
	for(int c=1; strcmp(comando,"*")!=0; c++)
	{
		printf("Case %d: ", c);
		if(strcmp(comando,"Hajj")==0) printf("Hajj-e-Akbar\n");
		else printf("Hajj-e-Asghar\n");
		scanf("%s",comando);
	}

	return 0;
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/12577_:::_Hajj-e-Akbar/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		String linea;

		for(int c=1; !(linea=br.readLine()).equals("*"); c++)
		{
			System.out.print("Case "+c+": ");
			if(linea.equals("Hajj")) System.out.println("Hajj-e-Akbar");
			else System.out.println("Hajj-e-Asghar");
		}		
		//-----------------------------------------------------------
		br.close();
	}
}
